src/customConfigTests/src/test/kotlin/org/vaccineimpact/orderlyweb/customConfigTests/ReportPagePermissionTests.kt

```diff
@@ -0,0 +1,148 @@
+package org.vaccineimpact.orderlyweb.customConfigTests
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Before
+import org.junit.Test
+import org.openqa.selenium.By
+import org.openqa.selenium.support.ui.ExpectedConditions
+import org.vaccineimpact.orderlyweb.models.Scope
+import org.vaccineimpact.orderlyweb.models.permissions.ReifiedPermission
+import org.vaccineimpact.orderlyweb.test_helpers.giveUserGlobalPermission
+import org.vaccineimpact.orderlyweb.test_helpers.insertReport
+
+class ReportPagePermissionTests : SeleniumTest()
```

Comment:
Factored these out into their own class as `ReportPageTests` was getting long and also pulled out shared code into helper methods to make the tests more readable.